src/java.base/share/native/libjli/java.c

```diff
@@ -58,9 +58,16 @@
 
 
 #include <assert.h>
+#if defined(J9VM_OPT_CRAC_SUPPORT)
+#include <ctype.h>
+#include <sys/wait.h>
+#endif /* defined(J9VM_OPT_CRAC_SUPPORT) */
 
 #include "java.h"
 #include "jni.h"
+#if defined(J9VM_OPT_CRAC_SUPPORT)
+#include "j9cfg.h"
```

Comment:
This `include` can't be enclosed within `#if defined(J9VM_OPT_CRAC_SUPPORT)`, it has `#define J9VM_OPT_CRAC_SUPPORT`, otherwise the rest of the code won't be included.